languages.toml

```diff
@@ -868,7 +868,18 @@ indent = { tab-width = 2, unit = "  " }
 
 [[grammar]]
 name = "markdown"
-source = { git = "https://github.com/MDeiml/tree-sitter-markdown", rev = "ab15701d8f3f68aeb74e30573b7d669a6ef2a7ed" }
+source = { git = "https://github.com/MDeiml/tree-sitter-markdown", rev = "142a5b4a1b092b64c9f5db8f11558f9dd4009a1b", subpath = "tree-sitter-markdown" }
+
+[[language]]
+name = "markdown_inline"
+scope = "source.markdown_inline"
+injection-regex = "markdown_inline"
```

Comment:
How about `markdown.inline`?

Reply:
Sure. For all of those? I have to admit I don't really understand what `scope` is for.

Reply:
> For all of those?

Yeah but `injection-regex` should probably be `"markdown\\.inline"` to be exact.

They're like textmate scopes. Currently we use them for some language naming things like what to send to a Language Server as `language_id`.